### PR TITLE
docs(zh): fix translation error

### PR DESCRIPTION
**Pull request by zgayjjf**
https://github.com/vuejs/composition-api-rfc/commit/6e89f73afc4f9d6718e933a07b1eb7214af537f1

### CODE REVIEW

File: zh/api.md [hunk 1]
```diff
@@ -922,7 +922,7 @@ export default {
 
 ### `isProxy`
 
-检查一个对象是否是由 `reactive` 还是 `readonly` 方法创建的代理。
+检查一个对象是否是由 `reactive` 或者 `readonly` 方法创建的代理。
```
Comment (zgayjjf) [inline zh/api.md:925]: 说明：
原文是 `Check if an object is a proxy created by reactive or readonly`。
这里应该是 `或者`，两者并存；而不是 `还是`，只取其一。

File: zh/api.md [hunk 2]
```diff
@@ -1009,7 +1009,7 @@ console.log(isReactive(bar.foo)) // false
 
 - 当渲染一个元素数量庞大，但是数据是不可变的，跳过 Proxy 的转换可以带来性能提升。
 
-这些 API 被设计成高级是因为这种特性仅停留在根级别，所以如果你将一个嵌套的，没有 `markRaw` 的对象设置为 reactive 对象的属性，在重新访问时，你又会得到一个 Proxy 的版本，在使用中最终会导致**标识混淆**的严重问题：执行某个操作同时依赖于某个对象的原始版本和代理版本。
+这些 API 被认为是高级的，是因为这种特性仅停留在根级别，所以如果你将一个嵌套的，没有 `markRaw` 的对象设置为 reactive 对象的属性，在重新访问时，你又会得到一个 Proxy 的版本，在使用中最终会导致**标识混淆**的严重问题：执行某个操作同时依赖于某个对象的原始版本和代理版本。
```
Comment (zgayjjf) [inline zh/api.md:1012]: 说明：
be considered => 被认为
be designed => 被设计为

这里并不是被设计为，而是我们认为它是高级的

**Comment by zgayjjf** [comment]
It seems like Chinese version RFC have diverged with the English version a lot. It there any need to keep them sync? @yyx990803 
I'd like to take charge of that.

File: zh/index.md [hunk 3]
```diff
@@ -583,7 +583,7 @@ export default {
 
 - 命名空间冲突。Mixin 之间的 property 和方法可能有冲突，同时高阶组件也可能和预期的 prop 有命名冲突。
 
-- 性能方面，高阶组件和无渲染组件需要额外的有状态的组件实例，避免无谓的开销。
+- 性能方面，高阶组件和无渲染组件需要额外的有状态的组件实例，从而使得性能有所损耗。
```
Comment (zgayjjf) [inline zh/index.md:586]: 说明：
原翻译和英文原意不符。

File: zh/index.md [hunk 4]
```diff
@@ -829,9 +829,9 @@ class App extends Component<Props> {
 }
 ```
 
-使用 decorator 导致它依赖一个带有很大不确定性的 stage-2 提案，特别是现在 TypeScript 的实现方案已经完全不是与 TC39 提案相同步的。除此之外没有什么太好的办法能够将定义在 `this.$props` 上的 prop 类型声明暴露出来， 这使得难以支持 TSX。从而破坏 TSX 支持。另外用户也会假设他们可以通过 `@prop message: string = 'foo'` 为某些 prop 声明默认值，但技术上这其实并不可用。
+使用 decorator 导致它依赖一个带有很大不确定性的 stage-2 提案，特别是现在 TypeScript 的实现方案已经完全不是与 TC39 提案相同步的。除此之外没有什么太好的办法能够将定义在 `this.$props` 上的 prop 类型声明暴露出来， 这使得难以支持 TSX。从而破坏 TSX 支持。另外用户也可能会认为他们可以通过 `@prop message: string = 'foo'` 为某些 prop 声明默认值，但技术上这其实并不可用。
 
-加之目前还没有很好的办法利用上下文的类型标注来推导 class 方法的参数——这意味着传递给一个 class 的 `render` 函数不能基于 class 的其它 property 来推导参数类型。
+加之目前还没有很好的办法利用上下文的类型标注来推导 class 方法的参数——这意味着传递给 class 的 `render` 函数的参数，不能基于 class 的其它 property 来推导参数类型。
```
Comment (zgayjjf) [inline zh/index.md:834]: 备注：
原译文翻译错误

File: zh/index.md [hunk 5]
```diff
@@ -829,9 +829,9 @@ class App extends Component<Props> {
 }
 ```
 
-使用 decorator 导致它依赖一个带有很大不确定性的 stage-2 提案，特别是现在 TypeScript 的实现方案已经完全不是与 TC39 提案相同步的。除此之外没有什么太好的办法能够将定义在 `this.$props` 上的 prop 类型声明暴露出来， 这使得难以支持 TSX。从而破坏 TSX 支持。另外用户也会假设他们可以通过 `@prop message: string = 'foo'` 为某些 prop 声明默认值，但技术上这其实并不可用。
+使用 decorator 导致它依赖一个带有很大不确定性的 stage-2 提案，特别是现在 TypeScript 的实现方案已经完全不是与 TC39 提案相同步的。除此之外没有什么太好的办法能够将定义在 `this.$props` 上的 prop 类型声明暴露出来， 这使得难以支持 TSX。从而破坏 TSX 支持。另外用户也可能会认为他们可以通过 `@prop message: string = 'foo'` 为某些 prop 声明默认值，但技术上这其实并不可用。
```
Comment (zgayjjf) [inline zh/index.md:832]: 说明：
用户不会“假设”，用户只会“认为”、“以为”

File: zh/index.md [hunk 6]
```diff
@@ -765,7 +765,7 @@ const { x, y } = useMousePosition()
 
 一些用户会顾虑 `setup()` 的返回语句变得冗长，像是重复劳动。
 
-我们相信明确的返回语句对可维护性是有益的。它使我们能够显式地控制暴露给模板的内容，并作为追踪组件中被定义的模板 property 的起点。
+我们相信明确的返回语句对可维护性是有益的。它使我们能够显式地控制暴露给模板的内容，并作为起点，追踪模板中某个属性在组件哪里被定义的。
```
Comment (zgayjjf) [inline zh/index.md]: 说明：
原句略不顺畅

Comment (Jinjiang) [reply zh/index.md]: property 不必翻译为“属性”，因为这个词在中文里歧义太多了

File: zh/index.md [hunk 7]
```diff
@@ -118,7 +118,7 @@ watchEffect(() => {
 
 这正是 Vue 响应式系统的精髓所在了！当你在组件中从 `data()` 返回一个对象，内部实质上通过调用 `reactive()` 使其变为响应式。而模板会被编译为渲染函数 (可被视为一种更高效的 `innerHTML`)，因而可以使用这些响应式的 property。
 
-> `watchEffect` 方法和 2.x 中的 `watch` 组合式 API 同样提供了一个 `watch` 函数，行为与 2.x 选项完全一致。
+> `watchEffect` 和 2.x 中的 `watch` 选项类似，但是它不需要把被依赖的数据源和副作用回调分开。*组合式 API*同样提供了一个 `watch` 函数，其行为和 2.x 的选项完全一致。
```
Comment (Jinjiang) [inline zh/index.md]:
```suggestion
> `watchEffect` 和 2.x 中的 `watch` 选项类似，但是它不需要把被依赖的数据源和副作用回调分开。组合式 API 同样提供了一个 `watch` 函数，其行为和 2.x 的选项完全一致。
```

**Comment by Jinjiang** [comment]
> It seems like Chinese version RFC have diverged with the English version a lot. It there any need to keep them sync? @yyx990803
> I'd like to take charge of that.

I suggest to record the latest commit hash when you decide to translate the docs each time. And then mention the hash in your pull request. So next time you could compare the hash to master to see the new changes. Actually this is what I practiced for cn.vuejs.org. For example: https://github.com/vuejs/cn.vuejs.org/pull/1085

Thanks.

File: zh/index.md (hunk 7 above)
Comment (zgayjjf) [reply zh/index.md]: 这里我本意是表明这是一个专有名词（毕竟中文没有下划线也没有空格也没有大写来表示，一般使用斜体表示专有名词），后来发现其他地方都没特殊说明，那索性也不多此一举了。

**Comment by zgayjjf** [comment]
> > It seems like Chinese version RFC have diverged with the English version a lot. It there any need to keep them sync? @yyx990803
> > I'd like to take charge of that.
> 
> I suggest to record the latest commit hash when you decide to translate the docs each time. And then mention the hash in your pull request. So next time you could compare the hash to master to see the new changes. Actually this is what I practiced for cn.vuejs.org. For example: [vuejs/cn.vuejs.org#1085](https://github.com/vuejs/cn.vuejs.org/pull/1085)
> 
> Thanks.

Seems like a really good practice.

I have changed and commited the code as you suggest, please review again.

**Review by Jinjiang — APPROVED** [review]

**Pull request merged**